setup.py

```diff
@@ -7,7 +7,7 @@
 # Get the long description from the README file
 long_description = (here / "README.md").read_text(encoding="utf-8")
 
-ray_deps = ["ray[default]", "pandas"]
```

Comment:
What was need in this before?

Reply:
We used Ray initialization that is similar/about same to one in Modin. We removed that logic as part of #21.

Reply:
ok